shrike/compliant_logging/logging.py

```
@@ -104,7 +100,7 @@ class CompliantLogger(logging.getLoggerClass()):  # type: ignore
     https://github.com/python/cpython/blob/3.8/Lib/logging/__init__.py
     """
 
-    def __init__(self, name: str, use_aml_metrics: bool = True):
+    def __init__(self, name: str, use_aml_metrics: bool = False):
```

@felixnext : I have change the default value of `use_aml_metrics` from True to False so that all existing pipelines will not be affected after the aml metric logging feature is added into shrike.

---

Makes sense, in theory this should not have a big impact (as it will fail only with a warning log message if azureml-core is not available) and then write metrics to logs.